source/_integrations/generic_thermostat.markdown

```diff
@@ -52,7 +52,7 @@ unique_id:
   required: false
   type: string
 heater:
-  description: "`entity_id` for heater switch, must be a toggle device. Becomes air conditioning switch when `ac_mode` is set to `true`."
+  description: "`entity_id` for heater, must be for `switch` or `fan` entity. Becomes air conditioning control when `ac_mode` is set to `true`."
```

Comment:
This seems incorrect, the YAML configuration doesn't have this limitation in code? (Although it could use some validations).

../Frenck